### PR TITLE
Add Hebrew Language Support

### PR DESCRIPTION
**Pull request by iethree**
Many thanks to our[ POEditor community contributors](https://poeditor.com/join/project/ynjQmwSsGh)!

Screen Shot 2024-01-23 at 8 51 45 AM

### CODE REVIEW

**Comment by mazameli** [comment]
@iethree In some random spot-checking of this translation on POEditor, I ran across one spot where the Hebrew for `{0} schemas` was missing the `{` brace. If you can find and fix that here in place with a find-replace, that would be opportunistic — I've already fixed it in POEditor, but wanted to spare you having to pull strings down again and all that.

File: locales/he.po [hunk 1]
```diff
@@ -399,7 +399,7 @@ msgstr "מחק את {0} ?"
 
 #: frontend/src/metabase/admin/datamodel/components/ObjectRetireModal.jsx:38
 msgid "Saved questions and other things that depend on this {0} will continue to work, but this {1} will no longer be selectable from the query builder."
-msgstr "{שאלות שמורות ופעולות אחרות שתלויות ב {0] ימשיכו לעבוד, אבל {1} לא יהיה זמין לבחירה במחולל הדוחות"
+msgstr "שאלות שמורות ופעולות אחרות שתלויות ב {0} ימשיכו לעבוד, אבל {1} לא יהיה זמין לבחירה במחולל הדוחות."
```
Comment (mazameli) [inline locales/he.po:402]: Isn't this putting the period at the beginning of the sentence, since it's rtl?

Comment (mazameli) [reply locales/he.po:402]:
```suggestion
msgstr ".שאלות שמורות ופעולות אחרות שתלויות ב {0} ימשיכו לעבוד, אבל {1} לא יהיה זמין לבחירה במחולל הדוחות"
```

Punctuation goes at the end (left), like you can see in this example from a Hebrew newspaper: 
image

Comment (iethree) [reply locales/he.po]: My mind is just unable to parse multiline RTL. thanks!

Screen Shot 2024-01-26 at 2 48 24 PM

**Review by mazameli — APPROVED** [review]

**Review by imrkd — APPROVED** [review]

**Pull request merged**